lib/src/layer/polygon_layer/polygon_layer.dart

```diff
@@ -108,6 +110,12 @@ class PolygonLayer extends StatelessWidget {
   /// screen space culling of polygons based on bounding box
   final bool polygonCulling;
 
+  /// how much to simplify the polygons, in decimal degrees scaled to floored zoom
+  final double? simplificationTolerance;
+
+  /// use high quality simplification
```

Comment:
We should clearly define what high quality simplification is, and it's advantages/use-cases and disadvantages.